twisted/internet/udp.py

```diff
@@ -311,6 +311,18 @@ def write(self, datagram, addr=None):
 
 
     def writeSequence(self, seq, addr):
+        """
+        Write a datagram constructed from a L{list} of L{bytes}.
+
+        @param seq: The data that will make up the complete datagram to be
+            written.
+        @type seq: L{list} of L{bytes}
+
+        @type addr: L{tuple} containing L{str} as first element and L{int} as
+            second element, or L{None}
+        @param addr: A tuple of (I{stringified IPv4 or IPv6 address},
+            I{integer port number}); can be L{None} in connected mode.
+        """
```

Comment:
The `addr` docs seem like they're copied from `write` -- I'm not that familiar with the tools and doc format, but is there a convention or a way that would make it possible just point at other docs (here something like "see the addr parameter of write") instead of having to copy/paste? If not, and if this matches the current convention, no objections here.